libs/barista-components/micro-chart/src/micro-chart.ts

```diff
@@ -280,13 +289,12 @@ function hideChartAxis(options: DtChartOptions): DtChartOptions {
 function convertToDataPoints(
   seriesData: Array<
     | number
-    | [number, number]
-    | [string, number]
-    | [string, number, number]
-    | [number, number, number]
-    | DataPoint
+    | [number | string, number | null]
+    | null
+    | SeriesColumnDataOptions
+    | SeriesLineDataOptions
   >,
-): DataPoint[] {
+): Array<SeriesColumnDataOptions | SeriesLineDataOptions> {
```

Comment:
For consistency

```suggestion
): (SeriesColumnDataOptions | SeriesLineDataOptions)[] {
```

Reply:
I have changed this, where I could. Thanks.